hare/eligibility/oracle.go

```diff
@@ -190,7 +189,11 @@ func (o *Oracle) buildVRFMessage(ctx context.Context, layer types.LayerID, round
 	}
 
 	// marshal message
-	msg := vrfMessage{Beacon: v, Round: round, Layer: layer}
+	roundUint16 := uint16(round)
```

Comment:
we have this in hare

```
preRound     uint32 = math.MaxUint32
certifyRound uint32 = math.MaxUint32 >> 1
```

i think we need to change round signature everywhere, weird that tests didn't crash

Reply:
maybe we don't send vrf for preround and certify round

Reply:
but i still think that that round signature needs to be changed everywhere for safety

Reply:
i am not sure either. i decided to drop this part of the change. too risky to be included in a no-op PR